modules/trusted-firmware-m/CMakeLists.txt

```diff
@@ -95,6 +95,8 @@ if (CONFIG_BUILD_WITH_TFM)
     list(APPEND TFM_CMAKE_ARGS -DMCUBOOT_IMAGE_NUMBER=${CONFIG_TFM_MCUBOOT_IMAGE_NUMBER})
   endif()
 
+  list(APPEND TFM_CMAKE_ARGS -DTFM_DUMMY_PROVISIONING=${CONFIG_TFM_DUMMY_PROVISIONING})
```

Comment:
Why is this different from the line below ?

Won't CONFIG_TFM_DUMMY_PROVISIONING be undefined when disabled?

so we get

-DTFM_DUMMY_PROVISIONING=